int-test/script/gen-actual

```diff
@@ -41,7 +41,7 @@ REPL_INPUT
 
 echo "space before and after"
 $PLANCK <<REPL_INPUT
- 3 
```

Comment:
Probably an inadvertent change. I think there was a space after this `3` (it was provoking a case where the space would cause an `EOF` to be emitted from the reader which needed to be handled.)